src/Files.App/ValueConverters/StringToSolidColorBrushValueConverter.cs

```diff
@@ -1,20 +1,21 @@
-﻿using Microsoft.UI.Xaml.Data;
+﻿using CommunityToolkit.WinUI.Helpers;
+using Microsoft.UI.Xaml.Data;
 using Microsoft.UI.Xaml.Media;
 using System;
 using Windows.UI;
 
 namespace Files.App.ValueConverters
 {
-	public class ColorToSolidColorBrushValueConverter : IValueConverter
+	public class StringToSolidColorBrushValueConverter : IValueConverter
 	{
 		public object? Convert(object value, Type targetType, object parameter, string language)
 		{
 			if (null == value)
 				return null;
 
-			if (value is Color)
+			if (value is string)
 			{
-				Color color = (Color)value;
+				Color color = ColorHelper.ToColor((string)value);
```

Comment:
minor nit, but you can avoid this cast using pattern matching.
```cs
if (value is string colorString) 
{
    Color color = ColorHelper.ToColor(colorString);
}

Reply:
Is there a benefit to it or just neater code?

Reply:
just for consistency! this type of pattern matching to cast is predominant in the codebase